web/src/main/java/com/zebrunner/reporting/web/SocialController.java

```diff
@@ -18,11 +20,17 @@
 @ConditionalOnProperty(name = "twitter.enabled", havingValue = "true")
 public class SocialController extends AbstractController implements SocialDocumentedController {
 
+    private static final String ZEBRUNNER_NEWS = "zebrunner_news";
+
     private final TwitterService twitterService;
 
     @GetMapping("/twitter/timeline")
     @Override
-    public List<Tweet> getUserTimeline(@RequestParam("userName") String userName, @RequestParam(value = "pageSize", defaultValue = "20") int pageSize) {
+    public List<Tweet> getUserTimeline(@RequestParam(value = "userName", defaultValue = ZEBRUNNER_NEWS) String userName, @RequestParam(value = "pageSize", defaultValue = "20") int pageSize) {
+        // Verify allowed user names
+        if(!Arrays.asList(ZEBRUNNER_NEWS).contains(userName)) {
```

Comment:
Probably such kind of validation should happen on the service level?

Reply:
Is that common experience to make validation on service layer?

Reply:
I would like to leave validation logic at controller layer.

Reply:
Move validation to service please. That way you can make sure that whoever calls that service won't use usernames we don't allow. And yes, it is common approach nowadays :) Typically we try to keep layer responsible only for web-specific things - dto to domain conversion, request params processing, etc